### PR TITLE
chore(HTTPSidebar): update Feature-Policy to Permissions-Policy

### PR DESCRIPTION
**Pull request by chrisdavidmills**
## Summary

MDN content has updated `Feature-Policy` to `Permissions-Policy`. This PR updates the `HTTPSidebar` macro to point to the correctly-named docs.

This is part of the work specified at https://github.com/mdn/content/issues/22347.

### Problem

Updated incorrect links in HTTPSidebar.

### Solution

Updates link strings and URLs.

---

### Testing

Confirmed — tested in local yari install, sidebar works fine.

### CODE REVIEW

**Comment by Rumyra** [comment]
Hey @chrisdavidmills - I can't say why the second test is failing, but I believe the first is to do with signed commits https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification which is required for yari 👍

**Comment by caugner** [comment]
@chrisdavidmills To sign all your commits in this PR, you can run `git rebase upstream/main --exec 'git commit --allow-empty --amend --no-edit --sign'`

**Comment by caugner** [comment]
> I can't say why the second test is failing

The test is failing, because the pages haven't _actually_ been renamed yet in mdn/content (PR pending merge), so the macro doesn't find any pages below https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy.

File: kumascript/macros/HTTPSidebar.ejs [hunk 1]
```diff
@@ -262,14 +262,14 @@ var text = mdn.localStringMap({
     'Caching': 'HTTP 缓存',
     'CORS': 'HTTP 访问控制（CORS）',
     'Resources': 'HTTP 规范',
-    'Feature_policy': '特征策略',
+    'Permissions_Policy': '特征策略',
```
Comment (yin1999) [inline kumascript/macros/HTTPSidebar.ejs:265]: A suggestion on zh-CN translation:

```suggestion
    'Permissions_Policy': '权限策略',
```

Comment (caugner) [reply kumascript/macros/HTTPSidebar.ejs]: Thanks! 🙏

**Comment by chrisdavidmills** [comment]
> ```shell
> git rebase upstream/main --exec 'git commit --allow-empty --amend --no-edit --sign'
> ```

@caugner hi! I've just gotten round to trying this, and it gives me the error `fatal: invalid upstream 'upstream/main'`. I have got the `upstream` remote set to `git@github.com:mdn/yari.git`, which I think it correct, right?

**Comment by caugner** [comment]
> > ```shell
> > git rebase upstream/main --exec 'git commit --allow-empty --amend --no-edit --sign'
> > ```
> 
> @caugner hi! I've just gotten round to trying this, and it gives me the error `fatal: invalid upstream 'upstream/main'`. I have got the `upstream` remote set to `git@github.com:mdn/yari.git`, which I think it correct, right?

If `git remote -v` lists mdn/yari as upstream, maybe your upstream is not up-to-date, and `git fetch upstream` would fix that.

**Comment by chrisdavidmills** [comment]
> `git rebase upstream/main --exec 'git commit --allow-empty --amend --no-edit --sign'`

@caugner hrm, I wish I'd never tried this in the first place. Every time I run it, it takes me on this weird rebase dance, and now it looks like I've totally screwed up the PR by adding a load of stuff into it that wasn't supposed to be there.

**Comment by caugner** [comment]
@chrisdavidmills That's weird, but you can always use [`git reflog`](https://git-scm.com/docs/git-reflog) to find out what commit this branch was at before.

**Comment by caugner** [comment]
@chrisdavidmills Oh, I think what might have additionally happened here is that when you `git push`ed, it merged the remote branch in your local branch first. After an interactive rebase, you'll always need to force-push.

**Comment by chrisdavidmills** [comment]
@caugner thanks for the help; I've managed to get this back to a sensible state now!

Of course, the commit signing verification thing is still failing...I'll have another look at the steps for that and make sure I've done it correctly.

And it is worth noting that the Feature-Policy -> Permissions-Policy content changes have now been merged (https://github.com/mdn/content/pull/22439), so this change should start to make sense to the system soon.

**Comment by chrisdavidmills** [comment]
@caugner I tried this again locally:

> git rebase upstream/main --exec 'git commit --allow-empty --amend --no-edit --sign'

Went through the interactive rebase, fixed a merge conflict, and then forced-pushed at the end.

The PR is in a sensible state, but it still fails, and now I've got 7 unsigned commits rather than 5 ;-)

I realised that the reason my commits were not signing earlier is that I had forgotten to include the -S modifier on my commit commands...but in this case it wouldn't help, as there was nothing new to commit after the rebase, just force-push.

Since this is only a small update (one file), shall I just close it and try again with a fresh PR, making sure I sign it this time?

**Review by caugner — APPROVED** [review]

**Pull request merged**